indigo/release/Dockerfile

```diff
@@ -5,7 +5,6 @@ FROM osrf/ros:indigo-desktop
 MAINTAINER Dave Coleman dave@dav.ee
 
 # apt-commands are combined in single RUN statement with lists folder removal to reduce image size
-RUN apt-get update \
- && apt-get upgrade -y \
```

Comment:
It's customary to keep/use the packages from the parent image. If you upgrade, it defeats a lot of the purpose in the space saving benefits of building from a layered image, as now the resulting image would include a diff history of changed package files, which could get huge fast. If you do need to upgraded, or something breaks downstream, then generally it's about time to trigger any parent imges upstream to rebuild. 

For the **non release images**, I let that slide and left your update command. I figured getting the most recent ros environment for development purposes may be more important that being agile with images. **Although** if temporal reproducibility important to you in even the development source images, I'd encourage you omit the use of upgrade there as well. But this is a tradeoff you should think or experiment with.